packages/common/core-interfaces/.eslintrc.js

```diff
@@ -5,6 +5,7 @@
 
 module.exports = {
     "extends": [
-        "@fluidframework/eslint-config-fluid"
+        "@fluidframework/eslint-config-fluid",
```

Comment:
Apparently this package was missed when we did some refactoring earlier. Not really related to this PR, but since we're making a change here anyways, can we update this to:
```suggestion
       require.resolve("@fluidframework/eslint-config-fluid"),
```

Reply:
Otherwise, extensions used by the base config won't be resolved correctly unless this package takes all of the same dependencies directly.

We've taken this approach as a sort of workaround for limitations in how eslint resolves dependencies.